packages/blade/src/components/Link/Link/Link.tsx

```diff
@@ -17,6 +17,12 @@ type LinkCommonProps = {
   target?: string;
   accessibilityLabel?: string;
 
+  /**
+   * Determines if the associated content controlled by this button is expanded or not.
+   * Generally used along with `accessibilityControls` on web
+   */
+  accessibilityExpanded?: boolean;
```

Comment:
who shall be responsible to change the value of this?

Reply:
My bad, I forgot to cleanup this from Link. Now we're directly using `makeAccessible` on BaseLink component to pass the a11y props through. Cleaned up in #1296